pyaerocom/io/read_airnow.py

```diff
@@ -348,9 +377,12 @@ def _filedata_to_statlist(self, arrs, vars_to_retrieve):
             list of :class:`StationData` objects, one for each var and station.
 
         """
+        # doubling of RAM usage!
```

Comment:
Worth reconsidering?

Reply:
not now, would implicate a major rewrite

Reply:
Just to clarify: Polishing should be done when we convert this reader to the new plugin structure